src/main/java/org/fogbowcloud/saps/engine/core/scheduler/arrebol/DefaultArrebol.java

```diff
@@ -63,12 +65,13 @@ public JobResponseDTO checkStatusJobById(String jobId) throws GetJobException {
 	}
 
 	@Override
-	public String checkStatusJobString(String jobId) throws GetJobException {
+	public String checkStatusJobString(String jobId) throws GetJobException, ArrebolConnectException {
```

Comment:
As already stated in the Arrebol interface, apparently this method is dead, check if it is still used in any case, if you do not remove it